#!/bin/sh /usr/share/dpatch/dpatch-run
## 320-magic-add-pmenu.dpatch by Edward Betts <edward@debian.org>
##
## DP: New magic for pmenu (Closes: #38543).

@DPATCH@

diff -Naur file-4.20.orig/magic/Magdir/commands file-4.20/magic/Magdir/commands
--- file-4.20.orig/magic/Magdir/commands	2007-01-19 19:28:01.000000000 +0000
+++ file-4.20/magic/Magdir/commands	2007-04-03 13:21:02.000000000 +0000
@@ -54,3 +54,8 @@
 0	string		Zend\x00		PHP script Zend Optimizer data
 
 0	string		$!			DCL command file
+
+# Type: Pdmenu
+# URL:  http://packages.debian.org/pdmenu
+# From: Edward Betts <edward@debian.org>
+0	string	#!/usr/bin/pdmenu	Pdmenu configuration file text
